src/aga/score.py

```diff
@@ -62,6 +62,7 @@ class Prize:
     name: str
     criteria: PrizeCriteria
     score_info: ScoreInfo
+    hidden: bool
 
 
 @dataclass(frozen=True)
```

Comment:
To make lints pass, add `# pylint: disable=too-many-arguments`, plus a comment that says something about how this is an external API and users need all these options.